book/installation.rst

```diff
@@ -98,6 +98,17 @@ number as the second argument of the ``new`` command:
     # Windows
     c:\projects\> php symfony.phar new my_project_name 2.3.23
 
+If you want your project to be based on the last Symfony LTS version, pass
```

Comment:
@94noni I can confirm that `lts` will always be the most recent LTS version (2.3 today and 2.7 in a few months). This version switch will be transparent to the installer and it will be done immediately after the new LTS version is published.